interfaces/dbus/backend.go

```diff
@@ -55,6 +56,27 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 	return "dbus"
 }
 
+// setupHostDBusConf will ensure that we have a dbus configuration
+// that points to /var/lib/snapd/dbus/{services,system-services}. This
+// is needed for systems that re-exec snapd and do not have this
+// configuration as part of the packaged snapd.
+func setupHostDBusConf(snapInfo *snap.Info) error {
+	coreRoot := snapInfo.MountDir()
+	for _, conf := range []string{
+		"session.d/snapd.session-services.conf",
+		"system.d/snapd.system-services.conf",
+	} {
+		dst := filepath.Join("/usr/share/dbus-1/", conf)
```

Comment:
This is not using the root directory from `dirs.go` so it will be harder to test.